### DISCUSSION TITLE
Visualize thumbnail match

### DISCUSSION THREAD

**Discussion by niltonlk** [YuelongWu/feabas#10] [General]
I know we can render the fine alignment output as thumbnail alignment, but is there a way to render the thumbnail match results so that we can check how good thumbnail match was performed?
I also noticed  that there is a tool called `visualize_align_match_coverage.py` in feabas/tools but it would be more intuitive if we can see the aligned thumbnails as image stacks.

**Comment by YuelongWu** [comment]
I'm considering adding an independent thumbnail workflow. But before that happens, what you could do is to copy the thumbnail matches to the fine alignment match folder (they are of the same format). Then you can do optimization, then render the transformed thumbnails. To make optimization faster, you may want to make the mesh size (in the alignment configuration files) coarser to match the thumbnail resolution. And remember to move/delete the mesh/matches/tform folders before you start the fine alignment step.

**Reply by niltonlk** [reply]
Thanks for the tips! I will give it a try!
